docs/static/troubleshoot/ts-logstash.asciidoc

```diff
@@ -204,3 +204,26 @@ As the logging library used in Logstash is synchronous, heavy logging can affect
 *Solution*
 
 Reset the logging level to `info`.
+
+[[ts-pipeline-logging-json-duplicated-message-field]]
+==== Logging in json format can write duplicate `message` fields
+
+*Symptoms*
+
+When format of log is `json` and certain log events (for example errors from JSON codec plugin)
+contains two instances of the `message` field.
+
+*Solution*
+In `config/logstash.yml` enable the strict json flag:
+
```

Comment:
We should say what the effect of this flag is, and what the data will look like. An example of before and after would be good here.

Reply:
Fixed by 5852c4bdf836ec62a20c122bef925c6a8fc57cd6